### ISSUE TITLE
Broken Links - User Guide - Getting Started

### ISSUE THREAD

**Issue by skywalke34** [rundeck/docs#169]
**Describe the bug**
Numerous broken links
ALSO - The default port for the web interface is 4440?   trying to use localhost:4440 also does not work.  Should probably be localhost:8080 unless you are using security certificates, which should also be documented.

**SCREENSHOT**
![image](https://user-images.githubusercontent.com/11443283/50162517-cfbd8000-029b-11e9-81c6-73a16837819a.png)
**Source page**
https://docs.rundeck.com/docs/manual/getting-started.html

**Comment by gschueler** [comment]
yes, the default port is 4440 (rpm/debian, and launcher war install), usually only Tomcat install uses 8080
